### PR TITLE
Remove satellites, etc.

### PR DESCRIPTION
**Pull request by noahlitvin**

### CODE REVIEW

**Review by mjlescano — APPROVED** [review]

File: packages/core-contracts/contracts/interfaces/IBeacon.sol [hunk 1]
```diff
@@ -1,8 +0,0 @@
-//SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
-
-interface IBeacon {
-    function getImplementation() external view returns (address);
-
-    function upgradeTo(address newImplementation) external;
-}
```
Comment (ajsantander) [inline packages/core-contracts/contracts/interfaces/IBeacon.sol:8]: How is this related to satellites? I think beacons are still useful and we might use at some point. However, I do agree to remove anything not being used atm. So your call...

File: packages/core-contracts/contracts/mocks/proxy/BeaconProxyMock.sol [hunk 2]
```diff
@@ -1,17 +0,0 @@
-//SPDX-License-Identifier: MIT
```
Comment (ajsantander) [inline packages/core-contracts/contracts/mocks/proxy/BeaconProxyMock.sol:1]: Again, depends on wether you want to keep beacons around or not.

File: packages/core-modules/contracts/mixins/AssociatedSystemsMixin.sol [hunk 3]
```diff
@@ -14,6 +14,10 @@ contract AssociatedSystemsMixin is AssociatedSystemsStorage {
     bytes32 internal constant _KIND_ERC721 = "erc721";
     bytes32 internal constant _KIND_UNMANAGED = "unmanaged";
 
+    function _getSystemAddress(bytes32 id) internal view returns (address) {
```
Comment (ajsantander) [inline packages/core-modules/contracts/mixins/AssociatedSystemsMixin.sol:17]: Maybe `_getAssociatedSystemAddress` would be a better name (more verbose, but more communicative)

File: packages/core-modules/contracts/modules/NftModule.sol [hunk 4]
```diff
@@ -12,11 +12,6 @@ import "../storage/NftStorage.sol";
 import "../interfaces/INftModule.sol";
 
 contract NftModule is INftModule, ERC721Enumerable, NftStorage, InitializableMixin, OwnableMixin {
-    event Mint(address owner, uint nftId);
-
-    // ---------------------------------------
-    // Chores
-    // ---------------------------------------
```
Comment (ajsantander) [inline packages/core-modules/contracts/modules/NftModule.sol:19]: 😆

**Review by ajsantander — APPROVED** [review]
